unit-tests/algo/d2rgb/compare-to-bin-file.h

```diff
@@ -37,10 +37,10 @@ bool is_equal_approximetly<algo::k_matrix, algo::k_matrix>( algo::k_matrix fx, a
 {
     bool ok = true;
 
-    ok = compare_and_trace(dx.get_fx(), fx.get_fx(), "fx");
-    ok = compare_and_trace(dx.get_fy(), fx.get_fy(), "fy");
-    ok = compare_and_trace(dx.get_ppx(), fx.get_ppx(), "ppx");
-    ok = compare_and_trace(dx.get_ppy(), fx.get_ppy(), "ppy");
+    ok = ok && compare_and_trace(dx.get_fx(), fx.get_fx(), "fx");
+    ok = ok && compare_and_trace(dx.get_fy(), fx.get_fy(), "fy");
```

Comment:
This means we won't get a comparison of fy if fx already failed... intended?
If not, you may want to reverse the order of the &&.